relay.go

```diff
@@ -379,6 +382,9 @@ func (r *Relayer) handleCallReq(f lazyCallReq) error {
 	origID := f.Header.ID
 	destinationID := remoteConn.NextMessageID()
 	ttl := f.TTL()
+	if ttl > r.maxTimeout {
+		ttl = r.maxTimeout
```

Comment:
Do we also want to rewrite the ttl in the frame? So that the caller also receives the shorter timeout. If so, we should add a unit test for that case.